src/translations/nbTranslations.json

```diff
@@ -196,6 +196,7 @@
     "nva_tagline": "Nasjonalt vitenarkiv (NVA)",
     "nvi": "Norsk vitenskapsindeks (NVI)",
     "open": "Åpne",
+    "or": "eller",
```

Comment:
Skal ikke alle translate felt starte med stor bokstav, så fikser i18 casinga?

Reply:
Kjører text transform med `uppercase` når jeg bruker teksten, så det spiller ikke så stor rolle for meg. @elkebab , hva er praksis?

Reply:
Ville sagt "Eller" er mest naturlig ja, så er det opp til der hvor det brukes å ev. overstyre casing i ene eller andre retning

Reply:
Fikser